packages/react-components/react-drawer/src/components/DrawerOverlay/DrawerOverlaySurface/DrawerOverlaySurface.types.ts

```diff
@@ -9,7 +9,7 @@ export type DrawerOverlaySurfaceSlots = DialogSurfaceSlots;
 /**
  * DrawerOverlaySurface Props
  */
-export type DrawerOverlaySurfaceProps = ComponentProps<DrawerOverlaySurfaceSlots>;
+export type DrawerOverlaySurfaceProps = DialogSurfaceProps;
```

Comment:
can we add tests that would catch this regression whenever dialog changes its apis ?

Reply:
I'll add this in a follow up PR as the scope of those tests are bigger than this PR.